test/ps_inverse.jl

```diff
@@ -5,7 +5,10 @@
     for d in 1:5
         S = Nemo.MatrixSpace(T, d, d)
         for case in 1:20
-            M = 1+S([random_ps(T) for i in 1:d, j in 1:d])
+            M = S([random_ps(T) for i in 1:d, j in 1:d])
+            while isequal(StructuralIdentifiability.LinearAlgebra.det(M), 0)
```

Comment:
This is tricky! You really want the matrix of constant coefficients to be invertible, not the whole matrix. Because the determinant may by, say, t + O(t^2), then you cannot invert it staying in power series.

Reply:
ah, so only check `det` of coefficients? or both coefficients and the full matrix?